src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -489,58 +504,68 @@ private void ProcessPendingSubmissions()
                 }
 
                 var submission = _pendingSubmissions.Dequeue();
+                _window.SetActiveCode(submission.Input);
+                Debug.Assert(submission.Task == null, "Someone set PendingSubmission.Task before it was dequeued.");
+                submission.Task = SubmitAsync();
+                if (submission.Completion != null)
+                {
+                    // ContinueWith is safe since TaskCompletionSource.SetResult should not throw.
+                    submission.Task.ContinueWith(_ => submission.Completion.SetResult(null), TaskScheduler.Current);
```

Comment:
Is the comment indicating we don't need an exception filter outside ContinueWith, and we don't need to await the task?